.github/workflows/pre-commit.yml

```diff
@@ -23,3 +23,25 @@ jobs:
       - uses: pre-commit/action@v3.0.0
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  check-chart-version-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.0.0
+
+      - name: Check if Chart.yaml versions are bumped if relevant
+        run: |
```

Comment:
We should keep the action relatively simple and simply invoke `make check` or something of the sort. That would allow users to replicate locally and evolve the logic without touching the workflow.

Reply:
yeah I considered that .... I don't think I had a reason not to do it that way then I just didn't lol